yoshi/pkg/utils/constants/constants.go

```diff
@@ -0,0 +1,11 @@
+package constants
+
+const (
+	CalicoPlugin         = "calico"
+	OvsPlugin            = "ovs"
+	VMFinalizerName      = "teardownVM"
+	NetworkFinalizerName = "teardownNetwork"
```

Comment:
Finalizer names should be (or at least convention to be) a proper DNS name. It tends to include the group name of the controller/api. So something like this:
```suggestion
        VMFinalizerName      = "vm.plumber.k8s.pf9.io"
	NetworkFinalizerName = "network.plumber.k8s.pf9.io"
```